scripts/build-brig.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-oses="linux windows darwin"
+oses="darwin"
```

Comment:
Was this change intended?  We still want to build linux and windows brig binaries, for uploading to each GitHub release, correct?

Reply:
Whoa. That was a local hack that snuck in. Eagle eye @vdice.

Reply:
Fixed it. Force pushed https://github.com/brigadecore/brigade/pull/1075/commits/433a1ce2d5d19c435df4eccf86dc56ad4f721b83 to replace the commit where that hack snuck in. Thanks again.